war/src/main/js/plugin-manager-ui.js

```diff
@@ -0,0 +1,59 @@
+import pluginManagerAvailable from './templates/plugin-manager/available.hbs'
+import pluginManager from './api/pluginManager';
+
+var debounce = null;
+
+function applyFilter(searchQuery) {
+    clearTimeout(debounce);
+    // debounce reduces number of server side calls while typing
+    debounce = setTimeout(function () {
```

Comment:
you can bring in lodash.debounce and use it here

Reply:
thanks, done